rfcs/0015-release-manager.md

```diff
@@ -0,0 +1,79 @@
+---
+feature: release-manager-nixos
+start-date: 2017-07-18
+author: Robin Gloster (@globin)
+co-authors: Franz Pletz (@fpletz)
+related-issues: --
+---
+
+# Summary
+[summary]: #summary
+
+NixOS currently has no process for electing release managers (RMs). We propose to
+switch to a model with two RMs, where each RM SHOULD
+serve for a consecutive term of two releases. A new RM is appointed
+by the previous team for each new release.
+
+# Motivation
+[motivation]: #motivation
+
+Currently release managing in NixOS has mostly been done by individuals who
+volunteered and were then chosen by the last release manager. Over the last
+few releases a process has been established and
+[documented](https://nixos.org/nixos/manual/index.html#release-process).
+As this makes it easier to cut a release this role should be passed on
+regularly and not be held by a single individual over a longer time.
```

Comment:
Yes, when we thought of this we had @FRidh's arguments in mind. I think there can always be small deviations from the rule if necessary. For instance, if a release managers can't continue her duties due to illness or other unforeseen circumstances, other people like preferably previous release mangers should be able to take over. We didn't want to specify this in detail but only give a rough proposal how the position should be filled. The important thing for us is that we have a new release manager for every release and a previous release manager for support.